packages/Ecotone/src/AnnotationFinder/AnnotationResolver/AttributeResolver.php

```
@@ -84,8 +84,12 @@ public function getAnnotationsForProperty(string $className, string $propertyNam
 
                     try {
                         $carry[] = $attribute->newInstance();
-                    } catch (Error) {
-                        // Do nothing: it is an attribute targeting a parameter promoted to a property
+                    } catch (Error $e) {
+                        if (\preg_match("/Attribute \"(.*)\" cannot target property/", $e->getMessage())) {
+                            // Do nothing: it is an attribute targeting a parameter promoted to a property
+                        } else {
```

Can we also apply this to `TypeResolver#86`? It's other way around there and we ignore all exceptions.

---

Ha, you found it ! I knew it was done elsewhere.